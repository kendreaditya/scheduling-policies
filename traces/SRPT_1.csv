1,0,6
2,2,2
3,6,4
4,14,6
5,17,4
6,17,2
