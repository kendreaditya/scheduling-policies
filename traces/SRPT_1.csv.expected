1,8
2,4
3,12
4,22
5,26
6,19
